File: 5597.cpp
#include <iostream>
using namespace std;

int main()
{
    int arr[31] = {
        0,
    };
    int input;
    int n, m;
    arr[0] = 1;
    for (size_t i = 1; i < 29; i++)
    {
        cin >> input;
        arr[input] = 1;
    }
    int j;
    for (j = 1; j < 31; j++)
    {
        if (arr[j] != 1)
        {
            n = j;
            break;
        }
    }
    for (j = 1; j < 31; j++)
    {
        if (arr[j] != 1)
        {
            m = j;
        }
    }
    if (n < m)
    {
        cout << n << endl;
        cout << m;
    }
    else
    {
        cout << m << endl;
        cout << n;
    }

    return 0;
}

File: 5597.js
const fs = require("fs");
const filePath = process.platform === "linux" ? "/dev/stdin" : "./input.txt";
let input = fs.readFileSync(filePath).toString().trim().split("\n").map(Number);

let arr = [];
let ans = [];
for(let i = 0; i<=30; i++){
    arr.push(i);
    ans.push(i);
}

arr.map((i, idx) => {
    input.map((j)=> {
        if(i === j){
            ans[i] = -1;
        }
    })
})

let ansStr = "";

ans.map((i) => {
    if(i !== -1 && i !== 0) ansStr += `${i} `; 
});

console.log(ansStr.trim());
